src/main/java/org/cactoos/package-info.java

```diff
@@ -35,9 +35,5 @@
  * @since 0.1
  * @see <a href="http://www.cactoos.org">Project site www.cactoos.org</a>
  * @see <a href="https://github.com/yegor256/cactoos">GitHub repository</a>
- * @todo #1445:30min For consistency sake with Func, let's
```

Comment:
@baudoliver7 please introduce a todo (modify this one) to continue the work, since we also need some other envelopes right?

Reply:
@victornoel Ok, I did also envelopes for `Runnable` and `Callable`. for `BiProc`, its envelope is already done in #1532. So we can say ticket is now entirely resolved.